tools/doc-generator/parse/parser.go

```diff
@@ -449,7 +449,7 @@ func ReflectType(typ string) reflect.Type {
 		return reflect.TypeOf(0)
 	case "float":
 		return reflect.TypeOf(0.0)
-	case "list of string":
+	case "list of strings":
 		return reflect.TypeOf(flagext.StringSliceCSV{})
 	case "map of string to string":
```

Comment:
for consistency, should this now be `map of strings to strings`? (can be done in separate PR)

Reply:
(same for the other maps)

Reply:
In the cases I've found, the value is never a slice, so I don't think it should be plural. Am I missing anything?

P.S. Going to merge this PR, but let's continue this conversation and I will eventually address it in a follow up PR.

Reply:
yeah, it's not a slice, it's just that a `map[string]string` can also have multiple string keys and multiple string values within one map, that's why I figured the description `map of strings to strings` would be more appropriate as well.

Reply:
@osg-grafana Could you help me phrase this data type description? If I have a mapping string-to-string and I can have multiple of them, is "map of strings to strings" correct? Just double checking before doing the change.